build/bazelrc/cross.bazelrc

```diff
@@ -43,14 +43,10 @@ build:docker-sandbox --spawn_strategy=docker --strategy=Javac=docker --genrule_s
 build:docker-sandbox --define=EXECUTOR=remote
 build:docker-sandbox --experimental_docker_verbose
 build:docker-sandbox --experimental_enable_docker_sandbox
-build:docker-sandbox --crosstool_top=@rbe_ubuntu_clang//cc:toolchain 
-build:docker-sandbox --host_javabase=@rbe_ubuntu_clang//java:jdk 
-build:docker-sandbox --javabase=@rbe_ubuntu_clang//java:jdk 
-build:docker-sandbox --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8 
-build:docker-sandbox --java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8 
```

Comment:
These flags were deprecated in bazel 5.0.0. See https://github.com/bazelbuild/bazel/issues/7849